package.json

```diff
@@ -46,11 +46,12 @@
   },
   "dependencies": {
     "classnames": "2.2.6",
-    "cozy-client": "6.48.0",
+    "cozy-client": "8.7.1",
```

Comment:
> I also had to add cozy-doctypes dep to not
get errors

Do you think it is a bug that needs solving in cozy-client ?

Reply:
Actually I think it's cozy-ui that requires cozy-doctypes, because the `ContactPicker` (not used here) needs it (see https://github.com/cozy/cozy-ui/blob/master/react/ContactPicker/index.jsx#L5). It's a peer dep, but when you `import { something } from 'cozy-ui'`, all imports are executed and webpack tries to find cozy-doctypes, and fails.